quipucords/api/networkprofile/tests_networkprofile.py

```diff
@@ -193,6 +193,14 @@ def test_create_empty_credentials(self):
              'ssh_port': '22',
              'credentials': []})
 
+    def test_create_invalid_credentials(self):
+        """A random int is not a valid credential."""
+        self.create_expect_400(
```

Comment:
Okay, but I thought the return code was going to be 404 and not 400. Did it change?

Reply:
404 is not found.  Since we are operating on network profiles I would expect a 404 to refer to a missing network profile (not a parameter used to update it).  So if the network profile we intend to update is missing then 404.  But if we found the profile and the update bad is bad (incorrect host cred) its a bad body (400).

We are creating a new network profile its not possible to have a 404 since it doesn't exist yet.